configs/templates/ufs-weather-model-static/spack.yaml

```diff
@@ -20,8 +20,8 @@ spack:
   - zlib@1.2.11~shared
   - bacio@2.4.1
   - crtm@2.4.0~fix
-  - esmf@8.4.1+debug~shared
-  - esmf@8.4.1~debug~shared
+  - esmf@8.4.1+debug~shared+parallelio~pio
+  - esmf@8.4.1~debug~shared+parallelio~pio
```

Comment:
Let's take out the `~pio` because that variant doesn't exist for 8.4.1.

Reply:
Fixed. Please test if this works as expected.